src/core/symbology-ng/qgssymbollayerv2utils.cpp

```diff
@@ -3781,3 +3781,146 @@ QString QgsSymbolLayerV2Utils::fieldOrExpressionFromExpression( QgsExpression* e
   return expression->expression();
 }
 
+QList<double> QgsSymbolLayerV2Utils::prettyBreaks( double minimum, double maximum, int classes )
```

Comment:
this change (moving _prettyBreaks) to QgsSymbolLayerV2Utils could be split into it's own commit - that way we could push these changes straight away and help decrease the size of this PR